### DISCUSSION TITLE
Pip screen ?

### DISCUSSION THREAD

**Discussion by jejehamers** [cpvalente/ontime#908] [Q&A]
Hello everyone,

Just upgraded to V2 on an old system.. Pip view has been removed. 

Just a simple question here : is it complicated to bring it back on the V2 ? Or to custom build a view that recreate its style ?

Thanks ;)

**Comment by cpvalente** [comment]
Hi @jejehamers, thank you for reaching out.
We are actually in the process of moving towards v3, maybe you want to take a look into that?

The PiP was removed because there was some misunderstandings on what it was supposed to do. it was basically a backstage screen with space for an overlay. There are no plans to bring it back, but maybe you have an interesting use case?

Meanwhile, I see several users simply render the different views in their media servers / obs and composite a view that looks the way they want.
Recreating the PiP view is also very approachable in simple HTML + CSS + Javascript if you have access to someone with that skillset

**Reply by jejehamers** [reply]
Thanks for the answer I could probably make the view myself, but have no time for that at the moment. I still have the installer for the v1 so not a problem. I think
It’s sad to have this view removed, I’m currently using it in combination with obs to send the view via ndi to the backstage/dressing rooms tvs. It’s way simpler than feeding it through sdi and others means. 

Thanks. 

Hamers J
